ProcessesApi.Tests/V1/Services/SoleToJointServiceTests.cs

```diff
@@ -691,12 +690,38 @@ public async Task ProcessStateIsUpdatedOnTenureInvestigationTrigger(string tenur
             // Assert
             CurrentStateShouldContainCorrectData(
                 process, trigger, expectedState,
-                new List<string> { /* TODO when next trigger is implemented */ }
+                new List<string> { /*TODO when next trigger is implemented*/ }
             );
             process.PreviousStates.Last().State.Should().Be(SoleToJointStates.ApplicationSubmitted);
             VerifyThatProcessUpdatedEventIsTriggered(SoleToJointStates.ApplicationSubmitted, expectedState);
         }
 
+        [Theory]
+        [InlineData(SoleToJointFormDataValues.Appointment)]
+        //[InlineData(SoleToJointFormDataValues.Approve)]
+        //[InlineData(SoleToJointFormDataValues.Decline)]
```

Comment:
Why are these commented?

Reply:
They weren't required, as they are being tested in the test above, I've updated this test :)